configcatclient/configcatclient.py

```diff
@@ -242,7 +243,7 @@ def get_key_and_value(self, variation_id):
                 if variation_id == rollout_percentage_item.get(VARIATION_ID):
                     return KeyValue(key, rollout_percentage_item[VALUE])
 
-        self.log.error('Could not find the setting for the given variation_id: ' + variation_id)
+        self.log.error('Could not find the setting for the specified variation ID: \'%s\'.', variation_id, event_id = 2011)
```

Comment:
I would use `variation_id` here in the log message on python:
```suggestion
        self.log.error('Could not find the setting for the specified variation_id: \'%s\'.', variation_id, event_id = 2011)
```